### DISCUSSION TITLE
Send Txs batch using zkSync SDK or API

### DISCUSSION THREAD

**Discussion by seddik11** [zkSync-Community-Hub/zksync-developers#147] [SDKs] [answered]
### Environment

Testnet

### Acknowledgement

- [ ] I understand that issues are expected with the Go SDK as they are being updated.

### Issue Description

Hi, is there a way to send a batch of signed transaction on ZkSync ? the goal here is to reduce the gas fees.
Is there some sdk ? I found those APIs and Sdk, are they still valid ?

https://docs.zksync.io/api/sdk/js/providers/#submit-transactions-batch
https://docs.zksync.io/api/v0.1/#submit-txs-batch

### Expected Behavior

None

### Code Example

None

**Comment by MexicanAce** [comment, marked as answer]
Duplicate of https://github.com/zkSync-Community-Hub/zkync-developers/discussions/146
